packages/block-editor/src/components/url-input/index.js

```diff
@@ -310,6 +326,13 @@ class URLInput extends Component {
 		}
 	}
 
+	reset() {
+		// move focus to the input
+		this.inputRef.current.focus();
+		// reset our suggestions
```

Comment:
>  reset our suggestions

Minor: are we actually "resetting" though? Because calling with an empty string might trigger a `initialSuggestions` if the `showInitialSuggestions` prop is provided.

Reply:
Maybe I’m confused about the update suggestions. I’m trying to return them to the initial suggestions if the field is empty and initial suggestions is on. When the cleared text was URL-like, the manual URL suggestion lingers around when it should be cleared.

Reply:
Would you recommend editing the comment for clarification?

Reply:
Actually thinking about it, that's what happens in the `updateSuggestions` so maybe the clarification is fine. I was being OTT.

Reply:
I was being OTT.